### PR TITLE
Block is the default for all newly-created databases

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1708.surge.sh

Updated pages:
- [Copy a database store](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Create, start, and stop databases](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-internals/store-formats)
- [Migrate a database](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/tools/neo4j-admin/migrate-database)

**Review by emmaholmbergohlsson — COMMENTED** [review]
Aligned is still default on community

**Comment by emmaholmbergohlsson** [comment]
[Switch the default format of the format setting to be block format for enterprise](https://trello.com/c/hUl32TNn/10025-switch-the-default-format-of-the-format-setting-to-be-block-format-for-enterprise)

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 1]
```diff
@@ -27,9 +27,10 @@ Having dots (`.`) in the database names is not recommended.
 This is due to the difficulty of determining if a dot is part of the database name or a delimiter for a database alias in a composite database.
 ====
 
-In Neo4j, the default store format for all new databases is `aligned`.
+In versions previous to Neo4j 5.22, the default store format for all new databases is `aligned`.
+From Neo4j 5.22, `block` is the default format for all newly-created databases as long as they do not have the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] setting specified. +
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:31]: yes, but the whole page is about enterprise only

File: modules/ROOT/pages/database-internals/store-formats.adoc [hunk 2]
```diff
@@ -423,7 +423,7 @@ The following tables show the format and Neo4j version compatibility and the lim
 |===
 | Name | Store format version | Introduced in | GA from
 
-| `BLOCK_V1`
+| `BLOCK_V1` label:default[Default for newly-created databases from Neo4j 5.22]
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/database-internals/store-formats.adoc:426]: This section is for EE only.

File: modules/ROOT/pages/tools/neo4j-admin/migrate-database.adoc [hunk 3]
```diff
@@ -81,9 +81,10 @@ If not specified, the tool migrates the target database to the latest known comb
 |Enable verbose output.
 |===
 
+[role=label--enterprise-edition]
 [NOTE]
 ====
-The block format is introduced in Neo4j 5.14 and is only available in Neo4j Enterprise Edition.
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/tools/neo4j-admin/migrate-database.adoc:86]: the note is EE only, there is a label above

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1708.surge.sh

Updated pages:
- [Copy a database store](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Create, start, and stop databases](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-internals/store-formats)
- [Migrate a database](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/tools/neo4j-admin/migrate-database)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1708.surge.sh

Updated pages:
- [Copy a database store](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Create, start, and stop databases](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-internals/store-formats)
- [Migrate a database](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/tools/neo4j-admin/migrate-database)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1708.surge.sh

Updated pages:
- [Copy a database store](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Create, start, and stop databases](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-internals/store-formats)
- [Migrate a database](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/tools/neo4j-admin/migrate-database)

File: modules/ROOT/pages/backup-restore/copy-database.adoc [hunk 4]
```diff
@@ -203,9 +203,10 @@ If you go from `high_limit` to `standard` or `aligned`, there is no validation t
 |
 |===
 
+[role=label--enterprise-edition]
 [NOTE]
 ====
-The block format is introduced in Neo4j 5.14 and is only available in Neo4j Enterprise Edition.
+The block format is introduced in Neo4j 5.14 and from Neo4j 5.22, is the default format for all newly-created databases as long as they do not have the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] setting specified.
```
[the remark this answers is not in the document]
Comment (emmaholmbergohlsson) [reply modules/ROOT/pages/backup-restore/copy-database.adoc:209]: Okay :+1:

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (emmaholmbergohlsson) [reply modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:31]: Okay yeah then it's okay. I personally don't really think about what pages are enterprise and not when I read them, but I guess a user would now

**Review by emmaholmbergohlsson — APPROVED** [review]
Okay, with the pages already being enterprise only this looks reasonable to me

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1708.surge.sh

Updated pages:
- [Copy a database store](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/backup-restore/copy-database)
- [Create, start, and stop databases](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Store formats](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/database-internals/store-formats)
- [Migrate a database](https://neo4j-docs-operations-1708.surge.sh/operations-manual/5/tools/neo4j-admin/migrate-database)

**Pull request merged**